bundle/Resources/config/services.yml

```diff
@@ -20,7 +20,7 @@ services:
     netgen_remote_media.provider.cloudinary.gateway:
         class: Netgen\Bundle\RemoteMediaBundle\RemoteMedia\Provider\Cloudinary\Gateway\CloudinaryApiGateway
         calls:
-            - [initCloudinary, ['%netgen_remote_media.parameters.cloudinary.account_name%', '%netgen_remote_media.parameters.cloudinary.account_key%', "%netgen_remote_media.parameters.cloudinary.account_secret%", "%netgen_remote_media.parameters.use_subdomains%"]]
+            - [initCloudinary, ['%netgen_remote_media.parameters.cloudinary.account_name%', '%netgen_remote_media.parameters.cloudinary.account_key%', "%netgen_remote_media.parameters.cloudinary.account_secret%", "%netgen_remote_media.parameters.upload_prefix%", "%netgen_remote_media.parameters.use_subdomains%"]]
```

Comment:
Since you are setting container parameter from the config, in the provider namespace:
https://github.com/netgen/NetgenRemoteMediaBundle/pull/98/files#diff-a0508c39bf4d6dc68c919658b7b952eca07a1bb8299fe94375e3b54325a78749R50
, this parameter in argument:
%netgen_remote_media.parameters.upload_prefix%"
 should be changed to:
%netgen_remote_media.parameters.cloudinary.upload_prefix%"

Otherwise, changing value defined through semantic config will have no effect

Reply:
@hknezevic fixed in https://github.com/netgen/NetgenRemoteMediaBundle/commit/f96609395bf45a0cbd2f2a6cda2ed34c7de714bd